waspc/data/Generator/templates/server/src/auth/providers/config/github.ts

```diff
@@ -77,13 +77,11 @@ const _waspConfig: ProviderConfig = {
 
         return createOAuthProviderRouter({
             provider,
-            stateTypes: ['state'],
+            optionalStateTypes: [],
```

Comment:
Since the `state` is _always_ sent and we rely on that knowledge in the new hooks, I've refactored this a bit to reflect the fact that the `state` will be always present in the oAuthState object.